sql/v2/test/tck/like_expression.yaml

```diff
@@ -115,4 +115,11 @@ tests:
     result: false
   - name: With type coercion from bool (4)
     expression: "FALSE LIKE 'fal%'"
-    result: true 
+    result: true
+
+  - name: Invalid string literal in comparison causes parse error
+    expression: "x LIKE 123"
+    result: false
+    error: parse
+    eventOverrides:
+      x: "123"
```

Comment:
Isn't this file coming from the spec? Should this updated there first https://github.com/cloudevents/spec/blob/main/cesql/cesql_tck/like_expression.yaml ?

Reply:
Yeah, I just wanted to include the change here since it is catching any regressions to the bug this is fixing. I'll sync this back to the spec once this merges